File: Year02/2nd semester/MPP/Lab/mpp-proiect-java-AlexandruPaul21/ProjectWebInterface/src/common-module/model/Flight.ts
export class Flight {
  id: number | undefined;
  destination: string | undefined;
  departureDateTime: Date | undefined;
  airport: string | undefined;
  availableSeats: number | undefined;

  constructor(
    id: number | undefined,
    destination: string | undefined,
    departureDateTime: Date | undefined,
    airport: string | undefined,
    availableSeats: number | undefined
  ) {
    this.id = id;
    this.destination = destination;
    this.departureDateTime = departureDateTime;
    this.airport = airport;
    this.availableSeats = availableSeats;
  }
}

File: Year02/2nd semester/MPP/Lab/mpp-proiect-csharp-AlexandruPaul21/CSharpInterface/model/Flight.cs
namespace CSharpInterface.model;

public class Flight : Entity<long>
{
    public string Destination { get; set; }
    public DateTime DepartureDateTime { get; set; }
    public string Airport { get; set; }
    public int AvailableSeats { get; set; }

    public Flight(string destination, DateTime departureDateTime, string airport, int availableSeats)
    {
        Destination = destination;
        DepartureDateTime = departureDateTime;
        Airport = airport;
        AvailableSeats = availableSeats;
    }
}